### PR TITLE
Add option to be able to add dependencies to the wheel

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Added option ``--add-deps`` to be able to add new dependencies to the wheel.

* This PR should be merged after #23 

It is also related to #20 

cc: @scopatz

### CODE REVIEW

**Comment by scopatz** [comment]
OK, thanks for putting this in! I'll wait until #23 is merged.  There seems to be similar issues here too. if you want to fix this PR up as well.

**Comment by marcelotrevisani** [comment]
> OK, thanks for putting this in! I'll wait until #23 is merged. There seems to be similar issues here too. if you want to fix this PR up as well.

Sure, I already applied the same suggestions. thanks!

**Comment by marcelotrevisani** [comment]
Rebased from master

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,7 @@ parts/
 sdist/
 var/
 wheels/
+.idea/
```
Comment (marcelotrevisani) [inline .gitignore]: As I'm using ``PyCharm`` and the IDE creates this folder to hold their configuration files there, I added it to be excluded from ``git`` as well

Comment (scopatz) [reply .gitignore]: That's fine!

**Comment by scopatz** [comment]
This looks good to me; is it ready to merge?

**Comment by marcelotrevisani** [comment]
> This looks good to me; is it ready to merge?

Yes, it is! :)

**Comment by scopatz** [comment]
Awesome! Should I cut a release with these new features?

**Comment by marcelotrevisani** [comment]
> Awesome! Should I cut a release with these new features?

That would be awesome!

**Comment by scopatz** [comment]
OK great, will do.

**Pull request merged**